### PR TITLE
Add SimpleCov

### PR DESCRIPTION
**Pull request by kmahern**
#### What
Add simplecov

#### Ticket
https://dsdmoj.atlassian.net/browse/CACP-163

#### Why
To give details of test coverage

### CODE REVIEW

File: bin/rails [hunk 1]
```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+if ENV['RAILS_ENV'] == 'test'
+  require 'simplecov'
+  SimpleCov.start 'rails'
+  puts "required simplecov"
```
Comment (jsugarman) [inline bin/rails]: unneeded comment?

File: .gitignore [hunk 2]
```diff
@@ -37,3 +37,6 @@ Brewfile.lock.json
 .env
 .env.local
 .env.**.local
+
+# ignore simplecov output
+/coverage
```
Comment (jsugarman) [inline .gitignore]: also add to `.dockerignore` 
[example](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/.dockerignore#L21)

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+require 'simplecov'
```
Comment (jsugarman) [inline spec/spec_helper.rb]: I would put this is rails_helper instead as best to keep spec_helper slim.

also consider adding config to exclude files from coverage 
[example config](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/spec/rails_helper.rb#L15-L42)

File: .gitignore (hunk 2 above)
Comment (jsugarman) [reply .gitignore]: forget that, already in `.dockerignore`

File: spec/spec_helper.rb (hunk 3 above)
Comment (kmahern) [reply spec/spec_helper.rb]: 👍

File: bin/rails (hunk 1 above)
Comment (jsugarman) [reply bin/rails]: did not realise this from 
see https://github.com/colszowka/simplecov#getting-started

**Pull request merged**